plugin-flex-ts-template-v2/src/feature-library/send-audio-rec-file/custom-components/AudioRecorder/AudioRecorder.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { IconButton } from '@twilio/flex-ui';
+
+import AudioRecorderPanel from '../AudioRecorderPanel';
+
+interface AudioRecorderState {
+  showRecorder: boolean;
+}
+
+export interface OwnProps {
+  showRecorder: boolean;
+}
+
+export type Props = OwnProps;
+
+class AudioRecorder extends React.Component<Props, AudioRecorderState> {
+  state = {
+    showRecorder: this.props.showRecorder,
+  };
+
+  dismiss = () => this.setState({ showRecorder: false });
```

Comment:
Can we dismiss the recorder and clear the file whenever the agent sends the message?

Reply:
Im not sure ... 
We may be able to add a listener to the send message action. 
But we may have to register and control it somehow.

Reply:
I've found a way for achieving this using a listener, just as I had suspected. It turned out to be a valuable learning experience. 
Task completed.